src/syntax-tree/body.ts

```diff
@@ -0,0 +1,46 @@
+import { Statement } from "./ast";
+import { Module } from "./module";
+
+export function rebuildBody(bodyContainer: Statement | Module, fromIndex: number, startLineNumber: number) {
```

Comment:
Are we planning on having a class for this or just methods?

Reply:
Yes, we will have them in a separate class. I wasn't able to do that because of some huge dependency issues. So just wanted to leave them here to be easily recognized later.